BUILDING.md

```diff
@@ -148,6 +148,121 @@ an installation tree under `build/install`.
 
 If the build did not succeed, please see the next section.
 
+## Building with `conan`
```

Comment:
I didn't know what conan is. I think it would be nice to add a docs link closer to the beginning (maybe hyperlinked to its first mention) so users can familiarize themselves with the 3rd party pkg

(I initially read `canon` like James' docker thing haha)